_commands/tools/scp.md

```diff
@@ -0,0 +1,33 @@
+---
+---
+
+scp
+-------
+
+`scp` is used to make a _secure copy_ of files or directories to and from a remote host. Similar to "cp" except either the source or destination should be a remote host. You'll want to run this command from your local machine.
+
+~~~ bash
+$ scp source user@host:destination
+$ scp user@host:source destination
+~~~
+
+<!--more-->
+
+### Useful Options / Examples
+
+#### `scp -r`
+~~~ bash
+$ scp -r source dest
+~~~
+
+#### Break it down
+* The `-r` option asks `scp` to _recursively_ copy an entire directory
+
+#### `scp -c`
+~~~ bash
+$ scp -c source dest
+~~~
+
+##### Break it down
+* The `-c` option asks `scp` to select a _cipher_ to encrypt the data being transferred.
```

Comment:
If you have to select a cipher, you know enough to read the man page. Either pick a different, more common flag to dissect or remove it entirely, please.